cmd/insights/root.go

```diff
@@ -114,7 +114,7 @@ func preRun(cmd *cobra.Command, args []string) {
 
 	configHandler, err := os.Open(configFile)
 	if err == nil {
-		configContents, err := ioutil.ReadAll(configHandler)
+		configContents, err := io.ReadAll(configHandler)
```

Comment:
Cool, how did you found that `io/ioutil` was deprecated over `io`? Was it some automated tooling?

Reply:
I've seen the deprecation notice in the past, in the [ioutil pkg documentation](https://pkg.go.dev/io/ioutil), but was reminded about it by [golangci-lint](https://github.com/golangci/golangci-lint) which gets run in CI.